src/mrpro/data/AcqInfo.py

```diff
@@ -228,33 +244,49 @@ def spatialdimension_2d(data: np.ndarray) -> SpatialDimension[torch.Tensor]:
             user6=tensor(idx['user'][:, 6]),
             user7=tensor(idx['user'][:, 7]),
         )
-
+        user = UserValues(
+            tensor_2d(headers['user_float'][:, 0]),
+            tensor_2d(headers['user_float'][:, 1]),
+            tensor_2d(headers['user_float'][:, 2]),
+            tensor_2d(headers['user_float'][:, 3]),
+            tensor_2d(headers['user_float'][:, 4]),
+            tensor_2d(headers['user_float'][:, 5]),
+            tensor_2d(headers['user_float'][:, 6]),
+            tensor_2d(headers['user_float'][:, 7]),
+            tensor_2d(headers['user_int'][:, 0]),
+            tensor_2d(headers['user_int'][:, 1]),
+            tensor_2d(headers['user_int'][:, 2]),
+            tensor_2d(headers['user_int'][:, 3]),
+            tensor_2d(headers['user_int'][:, 4]),
+            tensor_2d(headers['user_int'][:, 5]),
+            tensor_2d(headers['user_int'][:, 6]),
+            tensor_2d(headers['user_int'][:, 7]),
+        )
+        physiology_time_stamps = PhysiologyTimestamps(
+            tensor_2d(headers['physiology_time_stamp'][:, 0]).double(),
+            tensor_2d(headers['physiology_time_stamp'][:, 1]).double(),
+            tensor_2d(headers['physiology_time_stamp'][:, 2]).double(),
+        )
         acq_info = cls(
             idx=acq_idx,
-            acquisition_time_stamp=tensor_2d(headers['acquisition_time_stamp']),
-            active_channels=tensor_2d(headers['active_channels']),
-            available_channels=tensor_2d(headers['available_channels']),
-            center_sample=tensor_2d(headers['center_sample']),
-            channel_mask=tensor_2d(headers['channel_mask']),
-            discard_post=tensor_2d(headers['discard_post']),
-            discard_pre=tensor_2d(headers['discard_pre']),
-            encoding_space_ref=tensor_2d(headers['encoding_space_ref']),
+            acquisition_time_stamp=tensor_2d(headers['acquisition_time_stamp']).double(),
```

Comment:
I don't think casting to double is a good idea here and for the PhysiologyTimestamps, because these parameters are very likely to be used in signal models leading than to the whole signal model being evaluated in double precision. I would probably leave it as int - this might be another indicator for users that these are not times in [s] but discrete timesteps

Reply:
we should come up with a solution here.
it it is int, it is quite easy to make the mistake  "acquisition_time_stamp*2.5" which will not work as expected..

also, if i am not mistaken, we would anyways assume it is in 2.5ms steps when we use it in signal models.
so maybe we can just convert it when loading from ismrmrd to seconds, assuming 2.5ms steps.
if there ever is another source for ismrmrd files that uses a different step, we should handle that case separately.

So i would change the documentation to 
"usually in seconds since midnight (Siemens). For other vendors, this will be the vendor time stamp times 2.5e-3"